index.html

```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Cahee</title>
+    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header-bock">
+      <div class="container">
+        <div class="header__wrapper">
+          <div name="home" class="header__top">
+            <div class="header__logo logo">
+              <a href="index.html" class="logo__link"
+                ><img src="images/Logo_header.png" alt="logo" class="logo__img"
+              /></a>
+            </div>
+            <div class="header__nav">
+              <ul class="nav">
+                <li class="nav__item nav__active">
+                  <a href="#" class="nav__link">Home</a>
+                </li>
```

Comment:
модифікатор краще так
```suggestion
                <li class="nav__item nav__item--active">
                  <a href="#" class="nav__link">Home</a>
                </li>
```